endpoints/schemas/commercialInstructionSchemas.yaml

```diff
@@ -0,0 +1,283 @@
+openapi: "3.0.0"
+info:
+  title: Commercial Instructions Schema
+  version: 4.0.0.0
+  description: Commercial Instructions Schema, Based on TIP 4.0.0 Working Draft
+  termsOfService: http://placeholderdomain.io/terms/,
+  contact: 
+    name: MIT
+    email: tip-mailing-list@tbd.com
+    url: http://placeholderdomain.io
+  license: 
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+paths: {} 
+components: 
+  schemas: 
+    commercialInstructionsRequestObject:
+      description: This object will contain the instructions from the creative or agency to the seller
+      type: object
+      required:
+        - transactionHeader  
+        - referenceId
+        - buyerlocation
+        - Advertiser
+        - Product
+        - startDate
+        - endDate
+        - MediaOutlet
+        - Instruction
+      properties:
+        transactionHeader:
+          $ref: 'commonSchemas.yaml#/components/schemas/transactionHeader'
+        referenceId:
+          description: Information grouping all reference ids and their source that are associated to the proposal
+          type: array # TODO any constraints?
```

Comment:
Array has no specified datatype for the items: https://swagger.io/docs/specification/data-models/data-types/#array